CMakeLists.txt

```diff
@@ -11,9 +11,13 @@ endif()
 
 option(DRAW_FPS "Draw FPS on the top left corner of the window." OFF)
 option(SYSTEM_LIBS "Use system libraries when available." ON)
+option(EMSCRIPTEN_LOAD_SDL_PORTS "Load SDL and SDL_mixer emscripten ports instead of compiling them" OFF)
+option(LINK_MPG123 "Link mpg123 statically to Julius instead of relying on a library." OFF)
+
 cmake_dependent_option(VITA_BUILD "Build for the PlayStation Vita handheld game console." OFF "NOT MSVC" OFF)
 cmake_dependent_option(SWITCH_BUILD "Build for the Nintendo Switch handheld game console." OFF "NOT MSVC; NOT VITA_BUILD" OFF)
 cmake_dependent_option(ANDROID_BUILD "Build for the Android environment." OFF "NOT MSVC; NOT VITA_BUILD; NOT SWITCH_BUILD" OFF)
+cmake_dependent_option(EMESCRIPTEN_BUILD "Build for the Emscripten environment." OFF "NOT VITA_BUILD; NOT SWITCH_BUILD; NOT ANDROID_BUILD" OFF)
```

Comment:
Should we start thinking about a TARGET_PLATFORM variable instead of one boolean for each platform?

Reply:
Check Discord 😉